src/Microsoft.DotNet.Analyzers.Compatibility/Fixes/MetaCodeFixProvider.cs

```diff
@@ -91,7 +117,22 @@ public override void Apply(Workspace workspace, CancellationToken cancellationTo
                                                 m.GetParameters().Length == 1 &&
                                                 m.GetParameters()[0].ParameterType == typeof(string));
 
-                startMethod.Invoke(null, new[] { FileName });
+                startMethod.Invoke(null, new[] { Url });
+            }
+        }
+
+        private sealed class OpenInBrowserPreviewOperation : PreviewOperation
+        {
+            public OpenInBrowserPreviewOperation(string url)
+            {
+                Url = url;
+            }
+
+            public string Url { get; }
+
+            public override Task<object> GetPreviewAsync(CancellationToken cancellationToken)
+            {
+                return Task.FromResult<object>(string.Format(Resources.BrowseToUrl, Url));
```

Comment:
nit of nitness: the other two format strings have the `FormatString` appended to their name, `BrowseToUrl` is the only one that doesn't.